### PR TITLE
{url,page,folder,domain,punk}

### PR DESCRIPTION
**Pull request by deadjdona**
Proper order of parameters in --scope

### CODE REVIEW

**Comment by Slokilla** [comment]
Thanks for clearing your previous merge request ! I think you can squash all this commits into one “Ordering scope parameters orders” commit and it will be good for review :)

File: doc/wapiti.1 [hunk 1]
```diff
@@ -34,7 +34,7 @@ TARGET SPECIFICATION:
 \fB\-\-data\fR \fIURL_ENCODED_DATA\fR
 .
 .IP "\(bu" 4
-\fB\-\-scope\fR {page,folder,domain,url,punk}
+\fB\-\-scope\fR {url,page,folder,domain,punk}
```
Comment (Slokilla) [inline doc/wapiti.1:37]:
```suggestion
\fB\-\-scope\fR {url,page,folder,subdomain,domain,punk}
```

File: doc/wapiti.ronn [hunk 2]
```diff
@@ -24,7 +24,7 @@ TARGET SPECIFICATION:
 
   * `-u`, `--url` <URL>
   * `--data` <URL_ENCODED_DATA>
-  * `--scope` {page,folder,domain,url,punk}
+  * `--scope` {url,page,folder,domain,punk}
```
Comment (Slokilla) [inline doc/wapiti.ronn:27]:
```suggestion
  * `--scope` {url,page,folder,subdomain,domain,punk}
```

File: doc/wapiti.1.html [hunk 3]
```diff
@@ -110,7 +110,7 @@ <h2 id="OPTIONS-SUMMARY">OPTIONS SUMMARY</h2>
 <ul>
 <li><code>-u</code>, <code>--url</code> <var>URL</var></li>
 <li><code>--data</code> <var>URL_ENCODED_DATA</var></li>
-<li><code>--scope</code> {page,folder,domain,url,punk}</li>
+<li><code>--scope</code> {url,page,folder,domain,punk}</li>
```
Comment (Slokilla) [inline doc/wapiti.1.html:113]:
```suggestion
<li><code>--scope</code> {url,page,folder,subdomain,domain,punk}</li>
```

**Review by Slokilla — CHANGES_REQUESTED** [review]

**Comment by bretfourbe** [comment]
@deadjdona, https://github.com/wapiti-scanner/wapiti/pull/558 is merged so I close this one, thanks for contributing

**Pull request closed without merging**